adapter-guide/develop-stix-adapter.md

```diff
@@ -0,0 +1,602 @@
+# Developing a new STIX-shifter adapter
+
+* [Introduction](stix-adapter-overview.md)
+* [Scenario](#scenario)
+* [Prerequisites](#prerequisites)
```

Comment:
The Prerequisites link doesn't work. I think it's because this actually links to the heading (denoted with the \#\#) which is Prerequisites for Developer A. The tags, such as {: \#scenario} don't seem to do anything and they are actually visible when viewing it as the markdown.